@xen-orchestra/web-core/lib/locales/en.json

```
@@ -269,6 +270,48 @@
   "networks": "Networks",
   "new": "New",
   "new-features-are-coming": "New features are coming soon!",
+  "new-vm": "New VM",
+  "new-vm.add": "Add a new VM",
+  "new-vm.template": "Template",
+  "new-vm.pick-template": "Pick template",
+  "new-vm.install-settings": "Install settings",
+  "new-vm.no-config": "No configuration",
+  "new-vm.ssh-key": "SSH key",
+  "new-vm.custom-config": "Custom config",
+  "new-vm.user-config": "User config",
+  "new-vm.network-config": "Network config",
+  "new-vm.iso-dvd": "ISO/DVD",
+  "new-vm.pxe": "PXE",
+  "new-vm.system": "System",
+  "new-vm.multi-creation": "Multi creation",
+  "new-vm.vm-name": "VM name",
+  "new-vm.tags": "Tags",
+  "new-vm.boot-firmware": "Boot firmware",
+  "new-vm.boot-firmware-uefi": "The boot firmware is UEFI",
+  "new-vm.boot-firmware-bios": "The template already contains the BIOS strings",
+  "new-vm.copy-host": "Copy host",
+  "new-vm.vm-description": "VM description",
+  "new-vm.affinity-host": "Affinity host",
+  "new-vm.memory": "Memory",
+  "new-vm.socketsWithCoresPerSocket": "{nSockets} sockets × {nCores} cores/socket",
+  "new-vm.vcpu": "VCPU",
```

I think it should be:
```suggestion
  "new-vm.vcpu": "vCPU",
```
And maybe we should also handle the plural.